arclightning_backend/src/splash.rs

```diff
@@ -0,0 +1,16 @@
+extern crate rand;
+use rand::Rng;
+use std::fs;
+use std::io;
+
+fn select_random_image(image_dir: PathBuf) -> io::Result<PathBuf>{
```

Comment:
I change my mind on the return type of this. Let's make it a `Result<PathBuf, io::Error>` (which is the same exact thing, but more verbose)